xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -395,11 +395,19 @@ Rail transport, Roads and Road transport
 				<xsd:element ref="AuthorityRef"/>
 				<xsd:element ref="OperatorRef"/>
 			</xsd:choice>
-			<xsd:element name="additionalOperators" type="transportOrganisationRefs_RelStructure" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Additional OPERATORs for LINE.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
+			<xsd:choice>
+				<xsd:element name="additionalOperators" type="transportOrganisationRefs_RelStructure" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>Additional OPERATORs for LINE (from v1.1, prefer  additionalTransportOrganisations)</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="additionalTransportOrganisations" type="transportOrganisationRefs_RelStructure" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>Additional TRANSPORT ORGANISATIONs for LINE (authority, operator). +v1.1</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element ref="GroupOfOperatorsRef" minOccurs="0"/>
+			</xsd:choice>
 			<xsd:element name="otherModes" type="modeRefs_RelStructure" minOccurs="0">
```

Comment:
@skinkie  so you want to have additionalOperators and additionalAuthorities and no choice?
This is fine for me as well.

Reply:
Actually, I want to know why additionalOperators is preferred over "multiple operators".